charts/nextcloud/templates/nginx-config.yaml

```diff
@@ -14,6 +14,11 @@
         # could take several months.
         #add_header Strict-Transport-Security "max-age=15768000; includeSubDomains; preload;" always;
 
+        include mime.types;
+        types {
+          application/javascript js mjs;
+        }
+
```

Comment:
Please move this down to the same location as it is in https://docs.nextcloud.com/server/latest/admin_manual/installation/nginx.html

Reply:
moved :+1: 
there are multiple mismatches i think. `root` is also in a different place than in the docs

Reply:
Yeah, as I said we need to align them again. It helps a lot though to already put some pieces in the right place.